Tasks/PublishCodeCoverageResultsV1/publishcodecoverageresults.ts

```diff
@@ -44,7 +44,11 @@ async function run() {
         } else {
 
             if (autogenerateHtmlReport) {
-                tempFolder = path.join(getTempFolder(), 'cchtml');
+                if (reportDirectory) {
+                    tempFolder = path.join(getTempFolder(), reportDirectory);
```

Comment:
reportDirectory is full path '/usr/test/abc/' which breaks in conjuction with `tmp` folder

1. to avoid overriding, we should be adding some random string/number (cchtmlXX) to `cchtml` so that multiple tasks can be used in single pipeline.

Reply:
Good idea!